sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/entity/Translation.java

```diff
@@ -0,0 +1,19 @@
+package com.gfg.sellercenter.translation.entity;
+
+import lombok.*;
+
+@Builder
+@Getter
+@Setter
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
what about to add @Table(name="RESERVATION") if we are reading the table from sc database

Reply:
This will never connect to the database. This "service"/package will call the graph service which will return the data which you need.